tubearchivist/docker-compose.yml

```
@@ -0,0 +1,48 @@
+version: '3.7'
+services:
+  app_proxy:
+    environment:
+      APP_HOST: tubearchivist_server_1
+      APP_PORT: 8000
+  server:
```

We could disable the proxy authentication here. Also added a newline in between for better readability.
```suggestion
      APP_PORT: 8000
      PROXY_AUTH_ADD: "false"

  server:
```